src/fileseq/filesequence.py

```diff
@@ -784,9 +784,11 @@ def getPaddingNum(chars):
         Raises:
             ValueError: if unsupported padding character is detected
         """
-        match = PRINTF_SYNTAX_PADDING_RE.match(chars)
+        match = PRINTF_SYNTAX_PADDING_RE.match(chars) or HOUDINI_SYNTAX_PADDING_RE.match(chars)
         if match:
-            return int(match.group(1))
+            paddingNumStr = match.group(1)
+            paddingNum = int(paddingNumStr) if paddingNumStr else 1
+            return max(paddingNum, 1)
```

Comment:
Is this just paranoid checking being added? The regex implies a match on a positive integer only, so the case of there being an empty string or negative number would imply a bad regex?

Reply:
I actually do see now that the printf format, and the added houdini format allow for capturing a value of 0. I supose the regex for those two patterns could technically be more specific in requiring `[1-9]\d+`

Reply:
This check is added so the following format can be supported: "img.%0d.exr" 
It's a weird one for sure, but supported by some applications (ffmpeg come to my mind).